api/docs/release.dox

```diff
@@ -169,6 +169,8 @@ compatibility changes:
    collect more stats.
  - Moves the immediate operand of the VEX encoded version of vextractps and vinsertps
    to be the first source.
+ - The GPR versions of the vmovd and vmovq AVX opcodes are no longer combined into a
```

Comment:
I would still vote for splitting out in some fashion the compat changes.

Reply:
I moved the comment in the commit message "This patch contains a minor compatibility change that now splits the existing AVX general register versions of vmovd/vmovq into the opcodes vmovd/vmovq." to the top.